samples/shields/x_nucleo_iks01a1/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _x-nucleo-iks01a1-sample:
+
+X-NUCLEO-IKS01A1: MEMS Interial and Environmental Multi sensor shield
+#####################################################################
+
+Overview
+********
+This sample enables all sensors of X-NUCLEO-IKS01A1 shield by reading
```

Comment:
This sample enables all sensors of a X-NUCLEO-IKS01A1 shield, and then
    periodically reads and displays data from the shield sensors:

    - HTS221: Temperature and humidity
    - LPS25HB: Atmospheric pressure
    - LIS3MDL: 3-axis Magnetic field intensity
    - LSM6DSL: 3-Axis Acceleration